SK.WebApp/src/store/session/sessionSlice.ts

```diff
@@ -0,0 +1,47 @@
+import { createSelector, createSlice } from "@reduxjs/toolkit";
+import { RootState } from '..';
+
+interface SessionState {
+  username: string;
+  token: string;
+  image: string;
+};
+
+const initialState : SessionState = {
+  username: "",
+  token: "",
+  image: "",
+};
+
+export const sessionSlice = createSlice({
+    initialState,
+    name: 'session',
+    reducers: {
+      saveLoginData: (state, action) => {
```

Comment:
1. `saveLoginData` name doesn't really show what we're doing in this action, maybe `setSession` could be a better name?
2. `action` is of type `any` - please type that argument.

Example:

```ts
export const sessionSlice = createSlice({
  initialState,
  name: 'session',
  reducers: {
    setSession: (state, action: PayloadAction<SessionState>) => {
      state = action.payload;
    },
  },
});
```

Reply:
I changed name to setSession, and add types.